source/developer/localization.md

```diff
@@ -16,10 +16,11 @@ To translate a language:
 
 #### Translations Updates
 
-1. *New* and *updated* strings are imported to the [Mattermost Translation Server](http://translate.mattermost.com) on a daily basis at **12:00 GMT**
-2. A new **Pull Request** is being made with the latest translations that reaches at least **Beta Quality** to the [Mattermost Platform Repo](https://github.com/mattermost/platform) every day at **22:00 GMT** without the need to do it on your own.
+1. **New** and **updated** strings are imported to the [Mattermost Translation Server](http://translate.mattermost.com) each day at **00:00 GMT**
+2. A new pull request with latest translations that reach at least Beta Quality is submitted to the [Mattermost Platform Repo](https://github.com/mattermost/platform) each Monday at 22:00 GMT.
+    - A pull request will also be submitted on the day of major feature complete and code complete to ensure latest translations are included in the release. An additional pull request may also be submitted for a release candidate.
```

Comment:
Because we are talking about two different days, propose adding an "s" to "day"

`A pull request will also be submitted on the days of major feature complete and code complete to ensure latest translations are included in the release. An additional pull request may also be submitted for a release candidate.`